src/services/decoders/ClpIrDecoder/index.ts

```diff
@@ -188,6 +172,26 @@ class ClpIrDecoder implements Decoder {
 
         return results;
     }
+
+    /**
+     * Formats all log messages by prepending a dayjs formatted timestamp to beginning or each
+     * message
```

Comment:
how about
```suggestion
     * Formats unstructured log events by prepending a formatted timestamp to each message.
```
 Day.js is an implementation choice and we want to "hide" it as much as possible.